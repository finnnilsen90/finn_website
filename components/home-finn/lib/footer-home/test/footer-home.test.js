const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Footer = require('../footer-home.js');

let newFooter = new Footer();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Footer).toBeDefined();
    })

})  
    
describe('<Footer />', () => {

    const wrapper = render(<Footer />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})

describe('link should apear', () => {
    const wrapper = render(<Footer />);
    const link = wrapper.find('.git_link');

    it('on mouse over', () => {
        expect(link).toMatchSnapshot()
    });
});