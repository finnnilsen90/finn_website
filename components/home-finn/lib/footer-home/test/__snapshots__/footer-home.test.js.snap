// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders components 1`] = `
<div
  class="footer-home_Container font"
>
  <div
    class="logo_cont"
  >
    <a
      href="https://github.com/finnnilsen90"
    >
      <img
        class="github_logo"
        src="test-file-stub"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/finn-nilsen-50815182/"
    >
      <img
        class="linkedin_logo"
        src="test-file-stub"
      />
    </a>
    <a
      href="mailto:fnilsen1051@gmail.com"
    >
      <img
        class="gmail_logo"
        src="test-file-stub"
      />
    </a>
  </div>
  <div
    class="link_cont"
  >
    <p
      class="git_link"
    >
      https://github.com/finnnilsen90
    </p>
    <p
      class="linkedin_link"
    >
      https://www.linkedin.com/in/finn-nilsen
    </p>
    <p
      class="gmail_link"
    >
      fnilsen1051@gmail.com
    </p>
  </div>
</div>
`;

exports[`link should apear on mouse over 1`] = `
<p
  class="git_link"
>
  https://github.com/finnnilsen90
</p>
`;
